ui/app/AppLayouts/Profile/Sections/SoundsContainer.qml

```diff
@@ -3,13 +3,18 @@ import QtQuick.Controls 2.13
 import QtQuick.Layouts 1.13
 import "../../../../imports"
 import "../../../../shared"
+import "../../../../shared/status"
 
 Item {
     id: soundsContainer
     width: 200
     height: 200
     Layout.fillHeight: true
     Layout.fillWidth: true
```

Comment:
Either the anchors below will throw warnings because this is a Layout, or you can remove those 2 lines as well